wger/manager/templates/set/formset.html

```diff
@@ -1,5 +1,50 @@
 {% load wger_extras %}
 {% load i18n %}
+
+<div class="form-group">
```

Comment:
@Manorlds-Eaglespark This is good work. Now a user can actually use drop sets to calculate weight change instead of manually inserting the values. Well done

Reply:
Thanks. @Rhytah